### DISCUSSION TITLE
Current workflow needs to be documented?

### DISCUSSION THREAD

**Discussion by SamikBose** [ADicksonLab/wepy#113] [Ideas]
Hi @salotz,

We were talking about the wiki page in our meeting couple of weeks back, where we can put up the most recent strategy of running simulations in wepy in DicksonLab. I have a zenodo repo which I used to present current workflow of running unbinding simulations to some research labs in India. Do you think it's a good idea to put those links somewhere in Wiki?

**Comment by salotz** [comment]
Hi @SamikBose,
I think Zenodo is a good way to get specific DOIs for particular artifacts (i.e. for publishing or reporting to grants etc.) but not a good means for documenting things.

Instead of a wiki we can share these kinds of things in the discussions here, so I would just post the links either here or in a new thread.

Wikis are difficult to keep current and be authoritative and I think a discussion forum has more utility since its clear its from a particular point in time and author.
